addon/lib/document.js

```diff
@@ -127,6 +127,21 @@ export default Base.extend({
       console.debug(`${label}:`, any);
       return any;
     });
+    documentJexl.addTransform("min", (arr) => Math.min(...arr));
+    documentJexl.addTransform("max", (arr) => Math.max(...arr));
```

Comment:
Those `arr`s need to be filtered, including a test for it

Reply:
Also, the backend implemented a `sum` function (totally useful transform!) so the JS should implement this as well.

```js
documentJexl.addTransform("sub", (arr) => {
  const nums = arr.filter((num) => typeof num === "number");
  return nums.reduce((num, base) => base + num, 0);
});

Also maybe we should add `avg` for the average?
```

Reply:
Agree, I've added both since we can benefit from both of them